src/browser/modules/Sidebar/styled.jsx

```diff
@@ -61,7 +61,74 @@ export const StyledDocumentActionLink = (props) => {
   const {name, ...rest} = props
   return (
     <StyledHelpItem onClick={props.onClick}>
-      <StyledDocumentText {...rest}>{props.type === 'play' ? <PlayIcon /> : <QuestionIcon />}&nbsp;{name}</StyledDocumentText>
+      <StyledDocumentText {...rest}>{props.type === 'play' ? <PlayIcon />
+        : <QuestionIcon />}&nbsp;{name}</StyledDocumentText>
     </StyledHelpItem>
   )
 }
+
+export const StyledList = styled.ul`
+  list-style-type: none;
+  margin: 16px 0;
+`
+export const StyledListItem = styled.li`
+  list-style-type: none;
+  margin: 8px 0px 8px ${props => props.isChild ? '16px' : '8px'};
+  cursor: pointer;  
+`
+export const StyledListHeaderItem = styled.li`
+  list-style-type: none;
+  cursor: pointer;  
+`
+export const StyledFavoriteText = styled.span`
+  font-family: ${props => props.theme.primaryFontFamily};
+  width: 182px;
+  color: #bcc0c9;
+  font-size: 13px;
+  display: inline-block;
+  max-height: 54px;
+  overflow: hidden;
+`
+export const FoldersButton = styled.button`
+  background: transparent;
+  border: none;
```

Comment:
Please remove outlines as well.
That should be done for all items, but I'll just comment on this one.

![oskar4j 2017-04-18 at 16 32 22](https://cloud.githubusercontent.com/assets/570998/25136824/551664a4-2456-11e7-8783-09844c286e9e.png)